src/coreclr/vm/corelib.h

```diff
@@ -259,11 +259,36 @@ DEFINE_FIELD(DELEGATE,            METHOD_PTR_AUX,         _methodPtrAux)
 DEFINE_METHOD(DELEGATE,             CONSTRUCT_DELEGATE,     DelegateConstruct,          IM_Obj_IntPtr_RetVoid)
 DEFINE_METHOD(DELEGATE,             GET_INVOKE_METHOD,      GetInvokeMethod,            IM_RetIntPtr)
 
-DEFINE_CLASS(INT128,               System,                 Int128)
-DEFINE_CLASS(UINT128,              System,                 UInt128)
+DEFINE_CLASS(DOUBLE_SPC,            System,                 Double)
```

Comment:
I had to add a suffix for the types here cause otherwise they caused build errors due to name conflicts. I used `_SPC` for `System.Private.CoreLib` but maybe there's something better here?